includes/class-linkbacks-mf2-handler.php

```diff
@@ -565,7 +568,7 @@ public static function get_representative_author( $mf_array, $source ) {
 			return $mf_array['rels']['author'];
 		}
 
-		return null;
+		return 'someone';
```

Comment:
If we add this, every following check will break and it will add 'someone' most of the time!